pkg/apis/scheduling/types.go

```diff
@@ -309,6 +335,10 @@ type QueueSpec struct {
 
 	// Guarantee indicate configuration about resource reservation
 	Guarantee Guarantee `json:"guarantee,omitempty" protobuf:"bytes,4,opt,name=guarantee"`
+
+	// If specified, the queue's scheduling constraints
+	// +optional
+	Affinity *Affinity `json:"affinity,omitempty" protobuf:"bytes,6,opt,name=affinity"`
```

Comment:
the json comment is un-necessary :)

Reply:
@k82cn in k8s, what is the function of using json tag?

Reply:
json tag is used for code generator, so public API need json tag; this file is used within scheduler, so json tag is un-necessary :)